parts/inc/mess

```diff
@@ -96,22 +94,23 @@ die_sv(pTHX_ SV *sv)
 #endif
 #endif
 
-#ifndef vmess
-#if { NEED vmess }
+#if ! defined vmess && { VERSION >= 5.4.0 }
```

Comment:
You could this block and all other below blocks wrap into one big `#if { VERSION >= 5.4.0 }` block, instead of adding ` && { VERSION >= 5.4.0 }` into each condition.

Reply:
I tried this, and it didn't work.  You're welcome to submit a PR after I've merged what I have